content/projects/gsoc/students.adoc

```diff
@@ -50,31 +81,203 @@ We highly recommend to make some contributions to the project while you work on
 and mentors will consider contributions and interactions with the community when processing applications.
 
-Useful links:
+[[UsefulLinks]]
+=== Useful links
+
+Here are a list of links to help you get started on participating in Jenkins and in coding for the Jenkins project,
+in increasing level of difficulty.
+
+* link:https://jenkins.io/project/governance/[Project Governance]
+* link:https://github.com/jenkinsci/jenkins/blob/master/CONTRIBUTING.md[Contributing to Jenkins] (for contributions to the core)
+* link:https://wiki.jenkins.io/display/JENKINS/Beginners+Guide+to+Contributing[Beginners Guide to Contributing]
+* link:https://wiki.jenkins.io/display/JENKINS/Plugin+tutorial[Plugin tutorial]
+* link:https://jenkins.io/doc/developer/tutorial/[Another plugin tutorial]
+* link:https://jenkins.io/blog/2017/08/07/intro-to-plugin-development/[Plugin Development Tutorials, Videos, and More]
+* link:https://jenkins.io/doc/developer/plugin-development/pipeline-integration/[Writing Pipeline-Compatible Plugins]
+* link:https://jenkins.io/doc/developer/extensions/[Extension Points]
+* link:https://wiki.jenkins.io/display/JENKINS/Hosting+Plugins[Releasing plugins]
+
+There is also a
+link:https://issues.jenkins-ci.org/issues/?jql=labels%20%3D%20newbie-friendly[list of newbie-friendly issues].
+
+Feel free to contact potential mentors and org admins if you need help with choosing a newbie friendly issue to tackle.
+
+Having a look at recent plugins can help you getting familiar with the coding style and the type of code we write:
+
+* link:https://github.com/jenkinsci/configuration-as-code-plugin[Configuration as Code plugin]
+* link:https://github.com/jenkinsci/code-coverage-api-plugin[Code Coverage API plugin]
+* link:https://github.com/jenkinsci/remoting-kafka-plugin[Remoting over Kafka plugin]
+* link:https://github.com/jenkinsci/external-workspace-manager-plugin/[External Workspace Manager plugin]
+
+== Student Selection process
+
+Once the application period is over, administrators and mentors make a decision on which proposal to accept
+based on the proposal submitted to the Google Summer of Code website.
+Only proposals submitted before the deadline to the Google Summer of Code website are considered.
+
+We understand students are anxious to know whether they are selected or not, but admins and mentors are
+bound to secrecy until Google announces the selection results.
+We will not discuss the selection with students until Google makes the announcement.
+
+== Congratulations, you have been selected... or not
+
+We thank all students who reach out to us during the application period.
+If you have not been selected link:https://google.github.io/gsocguides/student/being-turned-down[read this],
+there could be many reasons, and some are even outside of our control.
+Do not feel bad, we encourage you to stay with the community, and apply again next year.
+
+If you have been selected, the community bonding period starts immediately.
```

Comment:
Unlikely. The student needs some time to prepare. I would say "within 2 days after the announcement" or so, mentors are expected to reach out within this time

Reply:
```suggestion
If you have been selected, the community bonding period starts within 2 days after the announcement.
```